xcm/xcm-simulator/fuzzer/src/relay_chain.rs

```diff
@@ -0,0 +1,189 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
I wonder if these two mock files can be re-used from the examples, or vice-versa.

Reply:
I think that should be possible. Keeping them separate would allow for mock files specific to the fuzzer, which might be useful later.